modules/client/examples/03-tokenVoting-client/18-undelegate-tokens.ts

```diff
@@ -0,0 +1,50 @@
+/* MARKDOWN
+---
+title: Undelegate Tokens
+---
+
+## Undelegates yout token voting power from the delegatee address
+
+This is the same as delegating to your own address.
+*/
+
+import {
+  ContextPlugin,
+  UndelegateTokensStep,
+  TokenVotingClient,
+} from "@aragon/sdk-client";
+import { context } from "../index";
+
+// Instantiate the general purpose client from the Aragon OSx SDK context.
+// Instantiate a plugin context from the Aragon OSx SDK context.
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+
+// Create a TokenVoting client.
+const tokenVotingClient: TokenVotingClient = new TokenVotingClient(
+  contextPlugin,
+);
+
+const tokenAddress = "0x1234567890123456789012345678901234567890"
+
+const steps = tokenVotingClient.methods.undelegateTokens(tokenAddress);
+
+for await (const step of steps) {
+  try {
+    switch (step.key) {
+      case UndelegateTokensStep.DELEGATING:
```

Comment:
Shouldn't the alias have the same underlying value but with the corresponding key?
```suggestion
      case UndelegateTokensStep.UNDELEGATING:
```

Reply:
I had to modify the steps a bit, but now this works give it a look